src/components/Field/Field.tsx

```diff
@@ -0,0 +1,88 @@
+import styles from "./Field.module.css";
+import Card from "../Card/Card";
+import React from "react";
+import { initialImages } from "../../constants/initialImages";
+import { IOpenCardsItem, IPlayingFieldItem } from "../../utils/types";
+import { shuffleArray } from "../../utils/util";
+
+function Field() {
+  const [openCards, setOpenCards] = React.useState<IOpenCardsItem[]>([]);
+  const [field, setField] = React.useState<IPlayingFieldItem[]>([]);
+  const images = [...initialImages, ...initialImages];
+  shuffleArray(images);
+  React.useEffect(() => {
+    const newField = images.map((image) => {
+      return {
+        url: image,
+        flip: false,
+        off: false,
+      };
+    });
+    setField(newField);
+  }, []);
+  React.useEffect(() => {
+    console.log(openCards);
+    setTimeout(() => {
+      handleCards();
+    }, "1000");
+  }, [openCards]);
```

Comment:
<img width="264" alt="Screenshot 2023-06-03 at 15 55 31" src="https://github.com/rmxr/memo/assets/34838167/fef9556a-59d0-4624-ab0b-fd230a5e8ba4">

Eslint and TS would notify about warnings and errors. Pay attention to them